README.md

```diff
@@ -40,6 +40,12 @@ In the same conda environment pip install from the q2-makarsa github repo:
 pip install git+https://github.com/BenKaehler/q2-makarsa.git
 ```
 
+Install the python libraries needed for the community prediction:
+
+```
+pip install python-louvain
+pip install netneurotools
```

Comment:
neurotools is not conda installable (which will complicate installation). Can this be replaced by another package?

Reply:
@BenKaehler what is your input on this?